docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -18,6 +18,60 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## User experience updates in Kubernetes Monitoring
+
+<!-- Beverly Buchanan --> 
+
```

Comment:
We want to add a release date for all entries. If you keep these grouped together then just do them in the two spots where you've grouped them. If not, then you'll want to add them to to each entry after the heading and before the availability note with a space on either side.
```suggestion

<Date>

```